src/test/javascript/spec/component/shared/course-group.component.spec.ts

```diff
@@ -83,7 +85,7 @@ describe('Course Group Component', () => {
     });
 
     afterEach(() => {
-        jest.restoreAllMocks();
+        jest.clearAllMocks();
```

Comment:
What's the reason behind this change?

Reply:
I thought it had fixed some issue, but apparently the tests pass without this change. I removed the change.